js/tests/views/attachment_spec.js

```diff
@@ -1,124 +0,0 @@
-/**
- * @author Luc Calaresu <dev@calaresu.com>
- *
- * Mail
- *
- * This code is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License, version 3,
- * as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License, version 3,
- * along with this program.  If not, see <http://www.gnu.org/licenses/>
- *
- */
-
-
-define(['views/attachment',
-		'models/localattachment',
-		'models/attachment',
-		'radio'],
-	function(AttachmentView, LocalAttachment, Attachment, Radio) {
-
-	describe('AttachmentView', function () {
```

Comment:
Has it really been deleted? I don't see that change it the full diff and also GitHub lists your review comment as *outdated*.